src/main.ts

```diff
@@ -5,9 +5,9 @@ import { ClarkModule } from 'app/clark.module';
 import { environment } from '@env/environment';
 
 // Global rxjs imports
```

Comment:
Can we remove this comment if we're not importing anything under it?

Reply:
I missed this when I was looking through. I went back and removed it.